tests/integration/cartography/intel/aws/test_guardduty.py

```diff
@@ -172,3 +174,70 @@ def test_sync_guardduty_findings(mock_get_findings, mock_get_detectors, neo4j_se
     assert (
         len(findings) == 2
     ), f"Expected 2 HIGH+ severity findings, got {len(findings)}"
+
+
+@patch.object(
+    cartography.intel.aws.guardduty,
+    "get_detectors",
+    return_value=LIST_DETECTORS["DetectorIds"],
+)
+@patch.object(
+    cartography.intel.aws.guardduty,
+    "get_detector_details",
+    return_value=GET_DETECTORS,
+)
+def test_sync_guardduty_detectors(
+    mock_get_detector_details, mock_get_detectors, neo4j_session
+):
+    """
+    Test that GuardDuty detectors sync correctly and create proper nodes and relationships.
+    Tests outcomes, not implementation details.
+    """
+    boto3_session = MagicMock()
+    create_test_account(neo4j_session, TEST_ACCOUNT_ID, TEST_UPDATE_TAG)
+
+    # Act
+    sync_detectors_function(
+        neo4j_session,
+        boto3_session,
+        [TEST_REGION],
+        TEST_ACCOUNT_ID,
+        TEST_UPDATE_TAG,
+    )
+
+    # Test outcomes - verify detector nodes have correct properties
+    expected_detector_nodes = {
+        ("12abc34d567e8fa901bc2d34e56789f0", "ENABLED"),
+        ("98zyx76w543u2ty109sr8q76p54321o0", "ENABLED"),
+    }
+    actual_detector_nodes = check_nodes(
+        neo4j_session, "GuardDutyDetector", ["detector_id", "status"]
+    )
+    assert actual_detector_nodes == expected_detector_nodes
+
+    # Test outcomes - verify ARN construction is correct
+    detectors = neo4j_session.run(
+        "MATCH (d:GuardDutyDetector) RETURN d.detector_id as detector_id, d.arn as arn"
+    ).data()
+
+    for detector in detectors:
+        expected_arn = f"arn:aws:guardduty:{TEST_REGION}:{TEST_ACCOUNT_ID}:detector/{detector['detector_id']}"
+        assert (
+            detector["arn"] == expected_arn
+        ), f"ARN mismatch for detector {detector['detector_id']}"
```

Comment:
Remove this test. It's overly verbose and doesn't use the `check_nodes()` pattern